src/frontend/Navigation/ScreenGroups/TabBar/TrackingTabBarIcon.tsx

```diff
@@ -0,0 +1,49 @@
+import React, {FC} from 'react';
+import {StyleSheet, View} from 'react-native';
+import {TabBarIcon} from './TabBarIcon';
+import {useTracking} from '../../../hooks/tracks/useTracking';
+import {Text} from '../../../sharedComponents/Text';
+import {TabBarIconProps, TabName} from '../../types';
+import {useTrackTimerContext} from '../../../contexts/TrackTimerContext';
+import {useTabNavigationStore} from '../../../hooks/useTabNavigationStore.ts';
+
+export const TrackingTabBarIcon: FC<TabBarIconProps> = props => {
+  const {isTracking} = useTracking();
+  const {timer} = useTrackTimerContext();
+  const {currentTab} = useTabNavigationStore();
+
+  return (
+    <>
+      {isTracking && (
+        <View style={styles.runtimeWrapper}>
+          <View style={styles.indicator} />
+          <Text style={styles.timer}>{timer}</Text>
+        </View>
+      )}
+      <TabBarIcon
+        {...props}
+        isFocused={currentTab === TabName.Tracking}
```

Comment:
If suggestion in TabBarIcon.tsx is applied:

```suggestion
        focused={currentTab === TabName.Tracking}
```